spec/models/availability_spec.rb

```diff
@@ -39,10 +39,11 @@
     it 'sets status for previous availabilities with matching start and end times to Cancelled' do
       start_time = Time.now
       end_time = start_time + 2.minutes
-      first_availability = create(:availability, person: a_person, start_time: start_time, end_time: end_time, created_at: Time.now - 2.minutes)
+      first_availability = create(:availability, person: a_person, start_time: start_time, end_time: end_time)
       second_availability = create(:availability, person: a_person, start_time: start_time, end_time: end_time)
 
-      expect(first_availability.reload.status).to eq('Cancelled')
+      first_availability = first_availability.reload
```

Comment:
moving the reload into its own expression like this still hasn't fixed the issue.
Test is read on Travis but green on local machine.